.gitignore

```diff
@@ -7,3 +7,5 @@
 !/emscripten.patch
 !/.github
 !/.github/workflows/compile.yml
+!/build.json
+!/cores.json
```

Comment:
could we add a newline at the end of the file

Reply:
Done